benchmark/single-source/FloatingPointConversion.swift

```diff
@@ -0,0 +1,66 @@
+//===--- FloatingPointConversion.swift ------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
```

Comment:
```suggestion
// Copyright (c) 2020 Apple Inc. and the Swift project authors
```

Reply:
Yeah, we need to update the benchmark generator template.